Realm/RLMAccessor.mm

```diff
@@ -179,25 +179,34 @@ static inline void RLMSetValue(__unsafe_unretained RLMObjectBase *const obj, NSU
     }
 }
 
-static inline size_t RLMAddLinkedObject(RLMObjectBase *link,
-                                        __unsafe_unretained RLMRealm *const realm,
-                                        RLMCreationOptions options) {
+static inline RLMObjectBase *RLMGetLinkedObjectForValue(__unsafe_unretained RLMRealm *const realm,
```

Comment:
i.e. add `static inline RLMObjectBase *RLMGetLinkedObjectForValue(RLMRealm *realm, NSString *className, id value, RLMCreationOptions creationOptions) NS_RETURNS_RETAINED;` on the line before (GCC doesn't support marking the definition directly so clang gives a warning).